class-coblocks.php

```diff
@@ -105,32 +105,42 @@ private function includes() {
 			require_once COBLOCKS_PLUGIN_DIR . 'includes/class-coblocks-register-blocks.php';
 			require_once COBLOCKS_PLUGIN_DIR . 'includes/class-coblocks-generated-styles.php';
 			require_once COBLOCKS_PLUGIN_DIR . 'includes/class-coblocks-body-classes.php';
-			require_once COBLOCKS_PLUGIN_DIR . 'includes/class-coblocks-form.php';
+			// WP.com custom - START
+			// require_once COBLOCKS_PLUGIN_DIR . 'includes/class-coblocks-form.php';
+			// WP.com custom - END
```

Comment:
Let's get rid of all these, as discussed [here](https://github.com/Automattic/coblocks/pull/1#issuecomment-722295912).

Reply:
🤦🏻 I forgot to remove the comments from this specific file, apologies.